lib/fathead/sass/README.txt

```diff
@@ -0,0 +1,3 @@
+Dependencies:
+- Python 2.7.6
```

Comment:
can you please make a file called `.python-version` and add `2.7.6` as the content?

We're going to start using pyenv and this will help 👍 

```